src/tidecli/utils/file_handler.py

```diff
@@ -120,33 +122,47 @@ def create_task(task: TaskData, overwrite: bool, user_path: str | None = None) -
     """
     # Sets path to current path or user given path
     if user_path:
-        user_folder = Path.cwd() / user_path
+        user_folder_root = Path.cwd() / user_path
     else:
-        user_folder = Path.cwd()
+        user_folder_root = Path.cwd()
 
     # Add course path to create task path
-    user_folder = user_folder / Path(task.path).name / task.ide_task_id
+    if task.task_directory is not None:
+        user_folder = user_folder_root / task.task_directory
```

Comment:
Shouldn't this variable be called task_folder / task_directory instead of user_folder?

Reply:
task_directory is reserved for the attribute coming from tim and it has total different meaning.
This is special exactly for this used who has decided t place this tasks to this
directory (folder).